### ISSUE TITLE
Exercise Ordering

### ISSUE THREAD

**Issue by patricksjackson** [exercism/cpp#237]
I'm not sure if the exercise ordering makes much sense. The core exercises and the exercise ordering was setup in #187, and they may need to be updated or changed. (I know of at least a couple changes I want to see happen, which I'll list here).

We're looking for someone to take on this task, figuring out if the ordering makes sense now after it's been active for several months, and possibly reordering many of the exercises. This would work best if we got feedback from the mentors for the track, and possibly from current or recent students.

I can help and provide input, but I may not have time to take this on myself. And I haven't mentored or done any exercises on exercism.io since they launched the new version of the website, so I don't personally have great context for what would make sense.

The one thing I want to happen is to make Gigasecond and Meetup not part of the core track to help with #233. I may end up making just that change myself if necessary, but the rest of the exercises need to be looked at as well.

**Comment by KevinWMatthews** [comment]
I agree with @patricksjackson 's concerns about Gigasecond and Meetup - they become much more difficult without Boost libraries. They could be discouraging/blocking for students.

Gigasecond is currently the [third core exercise](https://github.com/exercism/cpp/blob/a5a0983b205304a714f25e457630d7c035ccd01c/config.json#L54) in the track.

Meetup looks to be an [extra exercise](https://github.com/exercism/cpp/blob/a5a0983b205304a714f25e457630d7c035ccd01c/config.json#L212) that is unlocked by Space Age.

Mentor feedback would be excellent. You are the ones with experience!

**Comment by NobbZ** [comment]
Before doing random changes to the track, it might be worth to ask Maud in the #project-track-anatomy channel on the slack, whether or not she can take you into her beta program.

She is developing some tool that should help with big track restructurings.

**Comment by KevinWMatthews** [comment]
@NobbZ Thanks! That's worth looking into.

**Comment by patricksjackson** [comment]
I'm setting up a list for the Catch conversions, and it might help here as well. This is the current list of exercises and how we have them organized.

Unlocked by nothing:
- acronym
- all-your-base
- binary
- hexadecimal
- isogram
- pascals-triangle
- robot-simulator
- scrabble-score

Core track and what they unlock
- hello-world
    - grade-school
    - hamming
    - nucleotide-count
    - rna-transcription
- leap
    - difference-of-squares
    - nth-prime
    - queen-attack
    - say
    - sieve
- reverse-string
    - allergies
    - gigasecond
    - prime-factors
    - robot-name
    - sum-of-multiples
- space-age
    - beer-song
    - clock
    - meetup
- triangle
    - atbash-cipher
    - binary-search
    - trinary
- grains
    - food-chain
    - phone-number
    - roman-numerals
    - series
- raindrops
    - bob
    - crypto-square
    - word-count
- pangram
    - anagram
    - binary-search-tree
    - bracket-push
    - etl
